extension/src/popup/components/sendPayment/styles.scss

```diff
@@ -1,4 +1,26 @@
-// TODO - separate style sheet
+:root {
+  --bottom-btn-margin: 24px;
+
+  --continue-btn-height: 2.9rem;
```

Comment:
created a ticket for better bottom button spacing: https://github.com/stellar/freighter/issues/408